components/form/pta/src/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export const BASE_CLASS = 'sui-FormPta'
+
+class FormPta extends Component {
+  /**
+   * Avoid iframe re-rendering
+   */
+  shouldComponentUpdate () {
+    return false
+  }
+
+  /**
+   * Send settings to iframe content form
+   */
+  sendSettingsToForm = ({ target: { contentWindow } }) => {
+    const { redirectOnErrorUrl, formUrl, enableDraft, redirectOnSuccessUrl } = this.props
+    const formSettings = {
+      enableDraft,
+      formUrl,
+      exitURLs: {
+        success: redirectOnSuccessUrl,
+        error: redirectOnErrorUrl
+      }
+    }
+
+    contentWindow.postMessage(formSettings, '*')
```

Comment:
should the targetOrigin be a prop for security reasons?

Reply:
👍